### PR TITLE
docs: update to 20.0.0.alpha5, fix TS errors

### PR DESCRIPTION
**Pull request by web-padawan**
When updating docs to Vaadin 20.0.0.alpha4 there was a following error:

```
[14:58:36][Step 2/2] [ERROR] [tsl] ERROR in /opt/agent/work/83e2cf1ab9bef258/frontend/demo/component/avatar/avatar-group-internationalisation.ts(50,11)
[14:58:36][Step 2/2] [ERROR]       TS2739: Type '{ anonymous: string; activeUsers: { one: string; many: string; }; }' is missing the following properties from type 'AvatarGroupI18n': joined, left
```

This is a PR to add missing properties. Please forgive me if I'm mistaken, as I used Google translate 🙈

### CODE REVIEW

**Review by jouni — APPROVED** [review]
Impressive translation skills 👍 😄

**Comment by Haprog** [comment]
I guess this should update the platform version at the same time to make it compile.

**Comment by Haprog** [comment]
`ERROR  Cannot install with "frozen-lockfile" because pnpm-lock.yaml is not up-to-date with package.json`

**Review by tomivirkki — APPROVED** [review]

File: frontend/themes/theme.js [hunk 1]
```diff
@@ -1 +1 @@
-export { applyTheme } from './docs/docs.generated.js';
+export { applyTheme } from 'Frontend/generated/theme-docs.generated.js';
```
Comment (tomivirkki) [inline frontend/themes/theme.js:1]: This needs the change also https://github.com/vaadin/docs/blob/latest/webpack.dspublisher.js#L94

Comment (web-padawan) [reply frontend/themes/theme.js:1]: This file no longer exists, and the `generated` folder is already excluded. I will remove that line altogether.

Comment (tomivirkki) [reply frontend/themes/theme.js:1]: It doesn't exist in the version control but it's generated by Vaadin on startup (not under `generated` but under the default project theme). Apparently the generated name was changed in the V20 alpha period from `[name].generated.js` to `theme-[name].generated.js`.

Comment (tomivirkki) [reply frontend/themes/theme.js:1]: > not under generated but under the default project theme

I didn't actually check if this is true with the new version. If this has also changed (the project theme -generated.js file also appears under `generated`) then the line could be removed, as you suggested.

**Pull request merged**